tests/posix/posix_spawn.c

```diff
@@ -33,6 +33,6 @@ void run_cmd(char *cmd)
 }
 
 int main() {
-    run_cmd("/usr/bin/true");
+    run_cmd("/bin/true");
```

Comment:
this command isn't the best option here, I fixed this already but forgot to upstream (sorry): https://github.com/ArsenArsen/mlibc/commit/3f224a3204d9151d374ea93c89928122e6e4c0ba